RELEASES.md

```diff
@@ -1,3 +1,101 @@
+Version 1.36.0 (2019-07-04)
+==========================
+
+Language
+--------
+- [The order of traits in trait objects no longer affects the semantics of that
+  object.][59445] e.g. `dyn Send + fmt::Debug` is now equivalvent to
+  `dyn fmt::Debug + Send`, where this was previously not the case.
+- [Non-Lexical Lifetimes are now enabled on the 2015 edition.][59114]
+
+Libraries
+---------
+- [`TryFromSliceError` now implements `From<Infallible>`.][60318]
+- [`mem::needs_drop` is now available as a const fn.][60364]
+- [`alloc::Layout::from_size_align_unchecked` is now available as a const fn.][60370]
+- [`String` now implements `BorrowMut<str>`.][60404]
+- [`io::Cursor` now implements `Default`.][60234]
+- [Both `NonNull::{dangling, cast}` are now const fns.][60244]
+- [The `alloc` crate is now stable.][59675] `alloc` allows you to use a subset
+  of `std` (e.g. `Vec`, `Box`, `Arc`) in `#![no_std]` environments if the
+  environment has access to heap memory allocation.
+- [`String` now implements `From<&String>`.][59825]
+- [You can now pass multiple arguments to the `dbg!` macro.][59826] `dbg!` will
+  return a tuple of each argument when there is multiple arguments.
+- [`Result::{is_err, is_ok}` are now `#[must_use]` and will produce a warning if
+  not used.][59648]
+
+Stabilized APIs
+---------------
+- [`VecDeque::rotate_left`]
+- [`VecDeque::rotate_right`]
+- [`Iterator::copied`]
+- [`io::IoSlice`]
+- [`io::IoSliceMut`]
+- [`Read::read_vectored`]
+- [`Write::write_vectored`]
+- [`str::as_mut_ptr`]
+- [`mem::MaybeUninit`]
+- [`pointer::align_offset`]
+- [`future::Future`]
+- [`future::from_generator`]
+- [`future::get_task_context`]
+- [`future::poll_with_tls_context`]
+- [`future::set_task_context`]
+
+Cargo
+-----
+- [Cargo will now produce an error if you attempt to use the name of a required dependency as a feature.][cargo/6860]
+- [You can now pass the `--offline` flag to run cargo with accessing the network.][cargo/6934]
+
+Clippy
+------
+There have been numerous additions and fixes added clippy, see [Clippy's 1.36.0 release notes][clippy-1-36-0] for more details.
+
+Misc
+----
+- [When rustdoc is running with `--test` flag, it will now set
+  `cfg(test)`.][59940] This will allow items behind `cfg(test)` to
+  be documented.
```

Comment:
I believe this is going to be reverted or significantly changed before the release: https://github.com/rust-lang/rust/pull/61199

Reply:
Yes, let's drop this from the release notes, it's likely that we're going to revert or change something here and I'd prefer to not mention it rather than say the wrong thing (and we can clarify in an update later on).